scheduler/src/cook/test/testutil.clj

```diff
@@ -392,7 +392,8 @@
      (loop []
        (when-not (pred)
          (if (< (.getTime (java.util.Date.)) (+ start-ms max-wait-ms))
-           (recur)
+           (do (java.lang.Thread/sleep interval-ms)
+               (recur))
```

Comment:
Interesting, this has just been broken for a long time?

Reply:
apparently. my code caused it to fail